### PR TITLE
Kettlemc

### PR DESCRIPTION
**Pull request by UeberallGebannt**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by UeberallGebannt** [comment]
Are there still changes requested?

File: minecraft_servers/kettlemc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  "direct_ip": "KettleMC.net",
+  "server_wildcards": [
+    "%.kettlemc.de",
+    "%.kettlemc.net",
+    "%.kettlemc.eu"
+  ],
+  "supported_languages": [
+    "de", 
+	"en"
+  ],
```
Comment (JakobBruening) [inline minecraft_servers/kettlemc/manifest.json:13]: Please fix formatting

File: minecraft_servers/kettlemc/manifest.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  "social": {
+    "web": "https://www.kettlemc.net",
+    "web_shop": "https://shop.kettlemc.net",
+    "instagram": "kettlemc_net",
+    "twitter": "KettleMC",
+    "discord": "https://discord.gg/f9P9HEj",
+    "youtube": "https://www.youtube.com/@LeStegii"
```
Comment (JakobBruening) [inline minecraft_servers/kettlemc/manifest.json:20]: Only server-accounts are supported.

File: minecraft_servers/kettlemc/manifest.json [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    "%.kettlemc.de",
+    "%.kettlemc.net",
+    "%.kettlemc.eu"
+  ],
+  "supported_languages": [
+    "de", 
+	"en"
+  ],
+  "social": {
+    "web": "https://www.kettlemc.net",
```
Comment (JakobBruening) [inline minecraft_servers/kettlemc/manifest.json:15]: Website not available 
Bildschirm­foto 2023-01-27 um 16 50 12

**Review by JakobBruening — CHANGES_REQUESTED** [review]

**Comment by JakobBruening** [comment]
Youtube link is still invalid

**Comment by UeberallGebannt** [comment]
Works fine for me, or is it because of the name?
grafik

**Comment by JakobBruening** [comment]
As I commented, only server-accounts are supported (not private/youtuber accounts). Only a youtube account by the server directly is supported.

**Comment by UeberallGebannt** [comment]
For some reason I didn't see your change requests before, sorry for that. Going to fix this now 😅

File: minecraft_servers/kettlemc/manifest.json (hunk 1 above)
Comment (UeberallGebannt) [reply minecraft_servers/kettlemc/manifest.json]: Fixed in latest commit

File: minecraft_servers/kettlemc/manifest.json (hunk 2 above)
Comment (UeberallGebannt) [reply minecraft_servers/kettlemc/manifest.json]: Fixed in latest commit

File: minecraft_servers/kettlemc/manifest.json (hunk 3 above)
Comment (UeberallGebannt) [reply minecraft_servers/kettlemc/manifest.json]: Fixed

**Review by JakobBruening — APPROVED** [review]
Thank you!

**Comment by UeberallGebannt** [comment]
Thanks for your time and help ^^

**Pull request merged**